src/Tools/SarifValidator/ConsoleUtil.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Globalization;
+
+namespace Microsoft.CodeAnalysis.Sarif.SarifValidator
+{
+    internal static class ConsoleUtil
```

Comment:
Why do people use 'Util'? is there a problem with adding three more letters to this term so that it comprises a complete word? :)